scripts/nipype_utils.py

```diff
@@ -53,19 +51,25 @@ def nifti_to_data(args, X):
 
 
 def average_nifti(args):
-    """Reads in all the nifti images and calculates their average
-    """
+    """Reads in all the nifti images and calculates their average"""
     state_ = args["state"]
     input_dir = state_["baseDirectory"]
     output_dir = state_["transferDirectory"]
 
-    covar_x, _ = parse_covar_info(args)
+    covar_x = parse_covar_info(args)
 
     appended_data = 0
     for image in covar_x.index:
-        image_data = nib.load(os.path.join(input_dir, image)).dataobj[:]
-        if np.all(np.isnan(image_data)) or np.count_nonzero(
-                image_data) == 0 or image_data.size == 0:
+        try:
```

Comment:
@rssk Please remove this try except. You probably added it for debugging. Keep the line inside `try` and remove everything else.